app/models/user.rb

```diff
@@ -9,4 +9,22 @@ class User < ApplicationRecord
   has_many :posts
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
+
+  has_many :friendships
```

Comment:
You are missing the other side of the friendship (inverse_friendship) association.